components/leo/components.js

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import '@brave/leo/web-components/button'
+import { setIconBasePath } from '@brave/leo/web-components/icon'
+
+setIconBasePath('chrome://resources/brave-icons')
```

Comment:
Perhaps this should be `//resources/brave-icons`, so that it works ok in `chrome-untrusted://x`

Reply:
Good call!